core/services/ardupilot_manager/main.py

```diff
@@ -24,17 +26,19 @@
 
 
 class InterceptHandler(logging.Handler):
-    def emit(self, record) -> None:  # type: ignore
+    def emit(self, record: LogRecord) -> None:
         # Get corresponding Loguru level if it exists
+        level: Union[int, str]
         try:
             level = logger.level(record.levelname).name
         except ValueError:
             level = record.levelno
 
         # Find caller from where originated the logged message
+        frame: Optional[FrameType]
```

Comment:
Yes, where the code that you are fixing was provided.
As general rule, fixing commit that was not merged yet should be done in the same commit and not in a new one. 
Otherwise you are creating unnecessary commits.

Reply:
Your comment is on the ardupilot-manager service, not on cable-guy.

Reply:
They are two different code bases.